frontend/sozisel-app/package.json

```diff
@@ -17,6 +17,7 @@
     "graphql": "^15.5.0",
     "i18next": "^20.1.0",
     "material-ui-dropzone": "^3.5.0",
+    "moment-timezone": "^0.5.33",
```

Comment:
Why did you go with moment? This lib is no longer maintainted and weights a ton.

Reply:
OK, I will remove it.